databricks/koalas/series.py

```diff
@@ -5388,6 +5388,103 @@ def argmin(self):
             scol_for(sdf, self._internal.data_spark_column_names[0]) == min_value[0]
         ).head()[0]
 
+    # TODO: Remove skip after dropping Python 3.5
+    def compare(self, other, keep_shape=False, keep_equal=False):
+        """
+        Compare to another Series and show the differences.
+
+        Parameters
+        ----------
+        other : Series
+            Object to compare with.
+        keep_shape : bool, default False
+            If true, all rows and columns are kept.
+            Otherwise, only the ones with different values are kept.
+        keep_equal : bool, default False
+            If true, the result keeps values that are equal.
+            Otherwise, equal values are shown as NaNs.
+
+        Returns
+        -------
+        Series
+
+        Notes
+        -----
+        Matching NaNs will not appear as a difference.
+
+        Examples
+        --------
+        >>> s1 = ks.Series(["a", "b", "c", "d", "e"])
+        >>> s2 = ks.Series(["a", "a", "c", "b", "e"])
+
+        Align the differences on columns
+
+        >>> s1.compare(s2).sort_index().sort_index()  # doctest: +SKIP
```

Comment:
Why skipped?

Reply:
Btw, please remove one of `.sort_index()`s. :)

Reply:
Sorry, my mistakes  😅

Thanks for the point out!

Reply:
Seems like these are not fixed yet?